libbeat/idxmgmt/ilm/config.go

```diff
@@ -70,8 +70,8 @@ var DefaultPolicy = common.MapStr{
 			"hot": common.MapStr{
 				"actions": common.MapStr{
 					"rollover": common.MapStr{
-						"max_size": "50gb",
-						"max_age":  "30d",
+						"max_primary_shard_size": "50gb",
+						"max_age":                "30d",
```

Comment:
urgh... MapStr and global vars. We should have introduced a function that returns the default policy. Can you change it? The function would also get the ES version as parameter. The ES version is already known to the index manager, which is called by the ES connection callback.

Reply:
Make sense I will make the change here.

Reply:
@ph I don't think you pushed that change yet but this thread got resolved.

Reply:
@rdner good point, I should have added a comment, I would prefer we push this change in, In the current state of beats and the move to Elastic-Agent I don't think we should add more time to remove the global vars and dealing with MapStr.